### PR TITLE
Effects pane header misaligned

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
Effects pane header misaligned

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3331285

# How Tested
Before:
MicrosoftTeams-image (27)

After:
Screenshot 2023-07-12 at 3 09 39 PM

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const sidePaneHeaderStyles: IStackItemStyles = {
   root: {
     lineHeight: '1.5rem',
     fontSize: '1.125rem',
-    padding: '0.75rem 0.75rem 0.75rem 0.25rem',
+    padding: '0.75rem 0.75rem 0.75rem 0.50rem',
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts:43]: Just to confirm this change is by design, the current figma's have it slightly out hence the current behavior:
image

Comment (mgamis-msft) [reply packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts:43]: Seeing the same thing in Figma. Did Alex P file this bug?

Comment (prabhjot-msft) [reply packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts:43]: That is reported by AlexP in the bug bash, let me double check

Comment (prabhjot-msft) [reply packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts:43]: I talked to Alex, the Figma is updated with the alignment fix

**Comment by mgamis-msft** [comment]
I thinnk we change the style of the VideoEffectPane component's root Stack like so:
image
so that we don't affect the alignment in the other side panes like participant pane as seen in the snapshot updates. This is what it looks like from my testing
image

**Review by mgamis-msft — APPROVED** [review]

**Review by carocao-msft — APPROVED** [review]

**Pull request merged**